protocol/test/forge/TempleTeamPaymentsFactory.t.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.4;
+
+import "forge-std/Test.sol";
+import "../../contracts/admin/TempleTeamPaymentsFactory.sol";
+
+contract TempleTeamPaymentsFactoryTest is Test {
+    //
+    uint256 testAllocationLength = 3;
+    uint256 testLastEpoch = 17;
+    address testUser = 0xa6E83976204969B278CcB5246EdB698Af6a0FAeF;
+
+    //
+    TempleTeamPaymentsFactory public factory;
+    IERC20 public temple = IERC20(0x470EBf5f030Ed85Fc1ed4C2d36B9DD02e77CF1b7);
+    address multisig = 0xe2Bb722DA825eBfFa1E368De244bdF08ed68B5c4;
+
+    function setUp() public {
```

Comment:
Are you setting fork-url on command line? that's okay, but i would recommend using `vm.createFork(MAINNET_RPC_URL);` so that others running the setup can easily get up to speed